security/pkg/k8s/controller/workloadsecret.go

```diff
@@ -80,6 +84,13 @@ const (
 	caCertID = "ca-cert.pem"
 	// caPrivateKeyID is the private key file of CA.
 	caPrivateKeyID = "ca-key.pem"
+
+	// The name of the ConfigMap in each namespace storing the CA cert of self-signed CA.
+	CACertNamespaceConfigMap = "istio-ca-ns-configmap"
```

Comment:
nit: don't add configmap to the name. We already know its a configmap

Reply:
Done.